packages/llm/src/utils/tools.ts

```diff
@@ -0,0 +1,59 @@
+import { Api, KnowChainId } from "@polkadot-agent-kit/common"
+import { convertAddress, toMultiAddress } from "@polkadot-agent-kit/core"
+import { MultiAddress } from "@polkadot-api/descriptors"
+import { ToolResponse, ChainNotAvailableError, InvalidAddressError } from "../types"
+
+// Utility function to generate tool_call_id
+export const generateToolCallId = (prefix: string): string => `${prefix}_${Date.now()}`
+
+export const getApiForChain = (
+  apis: Map<KnowChainId, Api<KnowChainId>>,
```

Comment:
```suggestion
  apis: Map<KnownChainId, Api<KnowChainId>>,
```

Please update other please where `KnowChainId` should be `KnownChainId`